convert2rhel/repo.py

```diff
@@ -39,8 +39,7 @@ def get_rhel_repoids():
     we preferably enable the RHEL EUS repoids as those provide security updates over two years, in comparison to 6 months
     in case of the standard non-EUS repoids.
     """
-
-    if system_info.corresponds_to_rhel_eus_release():
+    if system_info.eus_system:
```

Comment:
Not entirely needed, but since we simplified this to be a property, maybe we could simplify this line to be an inline if statement? something like: 


```suggestion
    repos_needed = system_info.eus_rhsm_repoids if system_info.eus_system else system_info.default_rhsm_repoids
```

Reply:
Sounds good to me!